cmd/boulder-wfe2/main.go

```diff
@@ -158,6 +158,13 @@ type Config struct {
 			// default rate limits.
 			Overrides string
 		}
+
+		// MaxNames is the maximum number of subjectAltNames in a single cert.
+		// The value supplied SHOULD be greater than 0 and no more than 100,
+		// defaults to 100. These limits are per section 7.1 of our combined
+		// CP/CPS, under "DV-SSL Subscriber Certificate". The value must match
+		// the CA and RA configurations.
+		MaxNames int `validate:"min=0,max=100"`
```

Comment:
Nit: I understand that the validation tag catches this, but `MaxNames` could be declared as a `uint` instead. A minor type changes would be needed in the `NewWebFrontEndImpl` constructor and `WebFrontEndImpl` struct.